python/ray/data/read_api.py

```diff
@@ -320,6 +323,54 @@ def read_datasource(
     )
 
 
+@PublicAPI(stability="alpha")
+def read_mongo(
+    uri: str,
+    database: str,
+    collection: str,
+    pipelines: List[List[Dict]],
+    schema: Schema,
+    **kwargs,
+) -> Dataset[ArrowRow]:
+    """Create an Arrow dataset from MongoDB for the given pipelines.
+
+    Examples:
+        >>> import ray
+        >>> ds = ray.data.read_mongo( # doctest: +SKIP
+        ...     uri=MY_URI,
+        ...     database=MY_DB,
+        ...     collection=MY_COLLECTION,
+        ...     pipelines=MY_PIPELINES,
+        ...     schema=MY_SCHEMA
+        ... )
```

Comment:
Can you provide concrete examples of URI etc., instead of the placeholders?

Reply:
Added.